Rakefile

```diff
@@ -7,7 +7,12 @@ end
 
 desc 'Tweet random pet.'
 task :tweet_pet do
-  CutePets.post_pet
+  CutePets.post_pet_twitter
+end
+
+desc 'Post random pet.'
```

Comment:
I don't think the word "post" is specific to facebook. Perhaps the rake naming convention could follow the class name "post_pet_facebook"